src/components/AccountDashboard/AccountDashboard.tsx

```diff
@@ -112,15 +112,55 @@ function ContributedList({ address }: { address: string }) {
   )
 }
 
-function MyProjectsList({ address }: { address: string }) {
```

Comment:
Splits previous `MyProjectsList` into two components: `OwnedProjectsList` and `CreatedProjectsList` which show projects Owned and Created by the wallet. Both now use the `useDBProjectsQuery()` hook, instead of a subgraph query.